site/documentation/content/admin-guide/amqp-adapter-config.md

```diff
@@ -27,6 +27,7 @@ The following table provides an overview of the configuration variables and corr
 | `HONO_AMQP_KEY_PATH`<br>`--hono.amqp.keyPath` | no | - | The absolute path to the (PKCS8) PEM file containing the private key that the protocol adapter should use for authenticating to clients. This option must be used in conjunction with `HONO_AMQP_CERT_PATH`. Alternatively, the `HONO_AMQP_KEY_STORE_PATH` option can be used to configure a key store containing both the key as well as the certificate. |
 | `HONO_AMQP_KEY_STORE_PASSWORD`<br>`--hono.amqp.keyStorePassword` | no | - | The password required to read the contents of the key store. |
 | `HONO_AMQP_KEY_STORE_PATH`<br>`--hono.amqp.keyStorePath` | no | - | The absolute path to the Java key store containing the private key and certificate that the protocol adapter should use for authenticating to clients. Either this option or the `HONO_AMQP_KEY_PATH` and `HONO_AMQP_CERT_PATH` options need to be set in order to enable TLS secured connections with clients. The key store format can be either `JKS` or `PKCS12` indicated by a `.jks` or `.p12` file suffix respectively. |
+| `HONO_AMQP_SNI`<br>`--hono.amqp.sni` | no | `false` | Set whether the server supports Server Name Indication. By default, the server will not support SNI and the option is `false`. However, if set to `true` then the key store format, `HONO_AMQP_KEY_STORE_PATH`,  should be either `JKS` or `PKCS12` indicated by a `.jks` or `.p12` file suffix respectively. |
```

Comment:
IMHO it would be helpful to add a section to `secure_communication.md` describing how SNI can be used with the protocol adapters and what the server side key stores need to contain in order to make it work. This can also be a reference to the vert.x documentation in order to avoid duplication. But since SNI it very commonly known, I think we should explain a little. Would you agree?

Reply:
Yes totally make sense, I will add.

Reply:
I added the documentation , please take a look and welcome if are any remarks !